osgi.specs/docbook/104/service.cm.xml

```diff
@@ -2464,6 +2464,30 @@ ConfigurationPermission[?application,<xref
     fails or terminates regularly. Registered synchronous listeners will be
     informed immediately when the change happens regardless of a
     coordination.</para>
+    <para>The intend of this integration is that multiple events are collapsed
+    into one as updating a Managed Service or Managed Service Factory might result
+    in services being updated or unregistered what can trigger other service changes
+    what might be a costly operation and therefore result in an unwanted intermediate
+    states until the system has setled again. Collapsing here means, that multiple
```

Comment:
Spelling: 

> an unwanted intermediate states

Correction:  an unwanted intermediate state

(s removed at the end)

Reply:
thanks good to know.